src/common/utility.js

```diff
@@ -360,7 +361,52 @@ export const unslugify = (slug) => {
         })
     }
 }
+export const removeSpecialCharacterUrl = (url) =>
+    url &&
+    slugify(url)
+        .replace(/\?+/g, '') // Replace question mark with empty value
+        .replace(/[/]/g, '-') //Replace '/' with single -
+
+export const queryParams = {
+    get: (key) => {
+        const params = new URL(document.location).searchParams
+        let param_values = {}
+        //To get the params from the url
+
+        if (typeof key === 'string') {
+            return params.get(key)
+        } else {
+            key.forEach((k) => {
+                param_values[key] = params.get(k)
+            })
+        }
+        return param_values
+    },
+    set: (objects) => {
+        // To set the params from the url
+        const url = new URL(location)
+
+        Object.keys(objects).forEach((k) => {
+            const value = objects[k]
+            url.searchParams.set(k, value)
+        })
 
+        return window.history.replaceState(null, null, url)
+    },
+    delete: (key) => {
+        //To delete the params from the url
+        const url = new URL(location)
+        if (typeof key === 'string') {
+            url.searchParams.delete(key)
+        } else {
+            key.forEach((k) => {
+                url.searchParams.delete(k)
+            })
+        }
+
+        return history.replaceState(null, null, url)
+    },
+}
```

Comment:
Is there any difference from the build in browser `new URLSearchParams`?

Reply:
we are using URL.searchParams which is same of URLSearchParams.